WikipediaUnitTests/MWKSectionHasTextDataTests.m

```diff
@@ -0,0 +1,67 @@
+//  Created by Monte Hurd on 9/22/15.
+//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
+
+#import "XCTestCase+MWKFixtures.h"
+#import <XCTest/XCTest.h>
+#import "MWKDataStore+TemporaryDataStore.h"
+#import "WMFTestFixtureUtilities.h"
+#import "SessionSingleton.h"
+
+#define HC_SHORTHAND 1
+#define MOCKITO_SHORTHAND 1
+
+#import <OCHamcrest/OCHamcrest.h>
+#import <OCMockito/OCMockito.h>
+
+@interface MWKSectionHasTextDataTests : XCTestCase
+@property SessionSingleton* session;
+@end
+
+@implementation MWKSectionHasTextDataTests
+
+- (void)setUp {
+    [super setUp];
+    self.session = [[SessionSingleton alloc] initWithDataStore:[MWKDataStore temporaryDataStore]];
+}
+
+- (void)tearDown {
+    [self.session.dataStore removeFolderAtBasePath];
+    [super tearDown];
+}
+
+- (MWKArticle*)getTestingArticle {
+    MWKTitle* title     = [MWKTitle titleWithString:@"Barack Obama" site:[MWKSite siteWithDomain:@"wikipedia.org" language:@"en"]];
+    MWKArticle* article = [self articleWithMobileViewJSONFixture:@"Obama" withTitle:title dataStore:self.session.dataStore];
+    [article save];
+    return article;
+}
+
+- (void)testArticleHasAtLeastOneSectionWithZeroLengthHTML {
+    // Ensure at least one section of article has zero length section html.
+    MWKArticle* article              = [self getTestingArticle];
```

Comment:
It looks like you're testing that the test fixture has at least one non-empty section.  It's good that you thought to verify this, but having it as a standalone test is misleading.  It would be clearer if this were an assertion w/in a test that  requires the fixture to be non-empty.  e.g.

```objc
- (void)testSomethingExpectedWithCachedArticle {
    MWKArticle* myArticle = [...];
    NSAssert(myArticle.isCached, @"This test requires the fixture to have a non-empty section!");
    // exercise myArticle and verify functionality
}
```